drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -258,6 +323,18 @@ local function device_init(driver, device)
     if device:get_field(IS_PARENT_CHILD_DEVICE) == true then
       device:set_find_child(find_child)
     end
+    local main_endpoint = find_default_endpoint(device)
+    for _, ep in ipairs(device.endpoints) do
+      if ep.endpoint_id ~= main_endpoint and ep.endpoint_id ~= 0 then
+        local id = 0
```

Comment:
I would also add a unit test for the case where a device type that is not in our device type map is found. In this case, we wouldn't subscribe to the unaccounted for device type's clusters

Reply:
Added in 8f9481d2a5eed52608b22e5cab66822305543559!